app/models/member.rb

```diff
@@ -230,6 +230,18 @@ def freshman?
     end
   end
 
+  def penultimate?
+    educations.any? do |education|
+      education.status == 'active' && 2.year.ago < education.start_date && !Study.find(education.study_id).masters
```

Comment:
This would also return true for freshmans

Reply:
maybe `&& not freshman?`